# Surgical Outcomes: Both SPECT/CT and 4D CT Non-localizing

### Objective

Evaluate **surgical cure rates in patients with both negative SPECT/CT and 4D CT scans**, assessing outcomes when imaging fails to localize disease.

### Data Source

- `side_concordance_results.xlsx` containing pre-operative imaging results and surgical outcomes.

### Outputs

- Number and percentage of patients cured despite non-localizing imaging

- Interpretation notes for manuscript reporting
---


In [1]:
import pandas as pd 
preop_imaging_df = pd.read_excel('../../Data/final_cleaned_with_hyperplasia_concordance.xlsx')

In [2]:
# Define modalities
spect_modality = 'SPECT/CT'
modality_4dct = '4D CT Scan'

In [4]:
negative_spect_and_4d = preop_imaging_df[
    (preop_imaging_df['SPECT/CT'] == 'Non-localizing') &
    (preop_imaging_df['4D CT Scan'] == 'Non-localizing')
]
total_cases = negative_spect_and_4d.shape[0]
total_surgical_cure = negative_spect_and_4d[negative_spect_and_4d['Surgical Cure'] == 'Yes'].shape[0]
percentage = (total_surgical_cure / total_cases) * 100 

print(f"{total_cases} cases ({percentage:.2f}%) had both modalities non-localizing but were cured in surgery.")

negative_spect_4d_summary_df = pd.DataFrame({
    'Metric': [
        'Total cases with both modalities non-localizing',
        'Cases cured in surgery',
        'Percent cured within both-negative cases'
    ],
    'Count': [
        total_cases,
        total_surgical_cure,
        f"{percentage:.2f}%"
    ]
})
display(negative_spect_4d_summary_df)

16 cases (93.75%) had both modalities non-localizing but were cured in surgery.


Metric   Count
0  Total cases with both modalities non-localizing      16
1                           Cases cured in surgery      15
2         Percent cured within both-negative cases  93.75%

---

### Interpretation

- **Total cases:** 16 patients had **both SPECT/CT and 4D CT scans non-localizing**.

- **Surgical cure rate:** 15 out of 16 patients (**93.75%**) were cured at surgery despite negative imaging.

> **Clinical implication:** A high surgical cure rate (**~94%**) was achieved even when both imaging modalities were non-localizing, suggesting that **negative imaging does not preclude successful operative management**, likely due to strong intraoperative assessment and surgeon expertise.



### **Next Steps**

- Compare these findings with cure rates in patients who had positive imaging to guide **pre-operative imaging utility discussions**.